### DISCUSSION TITLE
短影音紅利時代來臨，人人都有機會利用短影音迅速收獲曝光

### DISCUSSION THREAD

**Discussion by Valentinaa0214** [1101-GS4719-C/Python-Community-GS4719-C#1097] [General]
近期無論是在IG、Youtube還是臉書等社群媒體都出現了類似tiktok的短影音功能，簡易的操作(往下滑就可以看下一部)與節奏快速的內容讓我常常忍不住一部接著一部看，不知不覺就看了三、四個小時。
雖然這種形式的影音能過獲得大量的觀看數、流量，但比較難讓我在看完後留下比較深刻的印象。
大家喜歡這種新形式的短影音嗎？對此的看法如何？
https://www.kolradar.com/blog/trend-sharing/kol-marketing-trend/short-video-content/
https://www.thenewslens.com/article/152553

**Comment by Sylvie0601** [comment]
我覺得IG、Facebook有這種功能可以觀眾接觸到更多的影音創作者、網紅，對於這些創作者來說，是一個很好增加曝光率的時候，我自己也很喜歡滑IG的短影音，常常會一直看就忘了時間的流逝，甚至是看到一篇有趣的、可愛的影片就會想要放到限時動態上和大家分享，我覺得對於創作者來說，比起抖音只能用滑、搜尋來看影片，透過看到再分享給朋友這種方式我認為更能將創作者的東西給更多人看見。

**Comment by Chiway1229** [comment]
我自己覺得這種短語音很容易讓人想一直看下去，以影音創作者來說是非常好的策略
但是以觀賞者來說，可能就沒辦法留下深刻印象了

**Comment by alicia0725** [comment]
我自己有剪輯及發布影片的經驗，滿明確地使用短影音的形式，能夠收獲更高的觸及率、流量，但就我自己觀看影片的習慣及經驗，若不是一部長達五分種以上的影音，基本上非常容易看過就忘，有時候就算回想也沒辦法記得具體內容，這也跟人們對於事物的記憶下降有關連

**Comment by lyien1021** [comment]
我覺得隨著這種短影片的流行，我的耐心也變得越來越少，以前80集的偶像劇都能看2遍，現在只要超過16集我就不會點進去看了🤣所以能花1分鐘就看完的影片對我來說真的很棒哈哈哈

**Comment by Tzuhsuannnnn** [comment]
不論是IG reels, youtube 短影片或是中國的抖音、小紅書，短影片都是現在的流行，雖然我也是個不喜換看很長影片的人，但我認為這樣的流行不是件好事，人們在這種風潮下，變得越來越沒耐性了解整件事情，而是喜歡透過短短幾秒來了解事情。

**Comment by Miinteaa** [comment]
除了各個社群、影音平台都想要跟上短影音的潮流以外，其實這個趨勢也大大影響了傳統影音產業。我曾經看過近年一部台灣的網劇叫做「愛的奧特萊斯」，該劇特別的地方在於一集只有三分鐘、且以直式的畫面呈現，與以往我們習慣的戲劇畫面呈現方式有很大的不同，雖然創意的呈現設計令人耳目一新，但過短的時長能涵蓋的劇情資訊量、連貫性也降低很多，反而會令我出戲。
另外，我認為短影音潮流雖然能夠讓我們在短時間內接受各種不同的資訊，但這些資訊有可能並不全面，甚至是完全正確，很容易接收到錯誤的資訊，所以我們在看這些短影片的同時更應該培養自身的思考及媒體識讀能力。

**Comment by alstinlin** [comment]
我蠻喜歡看一些短片的，因為短片會把內容全部壓縮在幾分鐘甚至幾秒，省去冗長的開頭就會讓人比較想看下去，而且很適合用來打發時間，但缺點是容易讓人一部一部得接著看，一不小心就在上面花太多時間，另外我也覺得短片的內容確實比較難有記憶點

**Comment by juliaouo** [comment]
我認為短影片在時間不多的情況下是個好選擇，用零碎的時間放鬆或娛樂，但若是在時間足夠的情況下，我更願意去看一些製作精良，品質較高、內容較豐富的長影片，能夠收穫的東西較多、記憶也會更加深刻，這是我的想法XD

**Comment by brian431** [comment]
我認為短影片的形式真的很容易讓使用者停不下來，對企業發展以及content creators來說是件好事。但我認為對整體人類的發展並不是件好事，畢竟大部分都真的太浪費時間了。

**Comment by ashleyy-huang** [comment]
我認為短影音真的害人不淺，常常一滑就是好幾個小時，然後再仔細回想時也不記得自己到滑了甚麼，或許這是一個可以加速曝光拿來賺錢的工具，但這對於使用者是不太好的的一個流行

**Comment by larry40305** [comment]
短影音形式的內容通常以迅速傳遞資訊、娛樂性強為特點，強調視覺效果和瞬間刺激。它們在迅速吸引觀眾的注意力和提供短暫的娛樂時刻方面表現出色，但可能在深度思考、知識傳遞和意識形態等方面缺乏充分的發展。

**Comment by aa103967622** [comment]
短影音這種東西就是很速食，它需要快速傳遞想要傳達給觀眾的資訊，所以相對來說它呈現的方式就是相當重點化，並且要足夠吸引人，未來，利用短影音宣傳的方式肯定會越來越多！
